admin/tests/unit/controllers/export-user-devices.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('User devices export controller', () => {
+  const { expect } = chai;
+
```

Comment:
```suggestion
describe('User devices export controller', () => {
  'use strict';
  
  const { expect } = chai;

```
Seems we are adding this everywhere.